packages/foam-vscode/src/utils.ts

```diff
@@ -184,3 +188,55 @@ export async function focusNote(notePath: string, moveCursorToEnd: boolean) {
     editor.selection = new Selection(range.end, range.end);
   }
 }
+
+export function getContainsTooltip(notes: Note[]): string {
```

Comment:
nice touch!

Reply:
@riccardoferretti actually here I feel it's missing some sort of sorting, as now it's just from the order of the provided array. A simple sorting would be alphabetical but ideally we'd have something based on maybe the number of note connections (for tags. for orphans we can keep it alphabetical I guess).

Reply:
Those are valid ideas, but I think it's ok to keep it simple for now - let's see once we start using the feature if we think it can be improved